setup.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
```

Comment:
setup.py file is needed to import the project as a python package - just install with:

pip install -e .

(added to readme as well)